float32 torque_desired
float32 speed_desired
float32 position_desired
float32 k_pos
float32 k_spd
